### PR TITLE
fix(http-add-on): Operator server uses the correct metrics port

### PR DESCRIPTION
**Pull request by JorTurFer**
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Related to https://github.com/kedacore/http-add-on/issues/812#issuecomment-1793412161

### CODE REVIEW

File: http-add-on/templates/operator/service.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
   ports:
   - name: https
     port: {{ default 8443 .Values.operator.port }}
-    targetPort: https
+    targetPort: metrics
```
Comment (tomkerkhove) [inline http-add-on/templates/operator/service.yaml:13]: Care to elaborate on this?

Comment (JorTurFer) [reply http-add-on/templates/operator/service.yaml:13]: Sure, there isn't any port named `https` in the operator:
https://github.com/kedacore/charts/blob/1931630189a5b57e5114e407d5b5be4354c27af0/http-add-on/templates/operator/deployment.yaml#L54-L58

And there is one occurrence where `.Value.operator.port` is used:
https://github.com/kedacore/charts/blob/main/http-add-on/templates/operator/deployment.yaml#L26

That's why I supposed that the target port is the metrics port

**Comment by JorTurFer** [comment]
Kindly reminder: PTAL @tomkerkhove xD

**Comment by JorTurFer** [comment]
I've included this PR on [this](https://github.com/kedacore/charts/pull/598)

**Pull request closed without merging**